zarr/__init__.py

```diff
@@ -9,7 +9,7 @@
                            zeros_like)
 from zarr.errors import CopyError, MetadataError
 from zarr.hierarchy import Group, group, open_group
-from zarr.n5 import N5Store
+from zarr.n5 import N5Store, N5FSStore
```

Comment:
This concerned me that it would need protecting by try/except block. In testing it, I realized FSStore only throws on `__init__` and therefore N5FSStore could be less conservative. I've pushed:

https://github.com/zarr-developers/zarr-python/pull/793/commits/aa4a723ecb9c776314f03f0a1abf14bacc10c215